src/style.css

```diff
@@ -1 +1,81 @@
 /* add styles here */
+:root {
+  --font-color: #3d4e61;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
+
+body {
+  margin: 8px;
+  font-family: Avenir;
+  font-weight: 300;
+  font-size: 16px;
+  color: var(--font-color);
+}
+
+.search-bar {
+  margin-top: 20px;
+}
+
+.search-bar__input {
+  box-sizing: border-box;
```

Comment:
it's better to set `box-sizing` for each element on the page (use `*` selector)

Reply:
I was looking for 1 hour to understand this behavior... ahhaha